connect/runtime/src/test/java/org/apache/kafka/connect/util/TopicAdminTest.java

```diff
@@ -568,18 +570,20 @@ private DescribeConfigsResponse describeConfigsResponseWithTopicAuthorizationExc
 
     private DescribeConfigsResponse describeConfigsResponse(ApiError error, NewTopic... topics) {
         if (error == null) error = new ApiError(Errors.NONE, "");
-        Map<ConfigResource, DescribeConfigsResponse.Config> configs = new HashMap<>();
+        List<DescribeConfigsResponseData.DescribeConfigsResult> results = new ArrayList<>();
         for (NewTopic topic : topics) {
-            ConfigResource resource = new ConfigResource(ConfigResource.Type.TOPIC, topic.name());
-            DescribeConfigsResponse.ConfigSource source = DescribeConfigsResponse.ConfigSource.TOPIC_CONFIG;
-            Collection<DescribeConfigsResponse.ConfigEntry> entries = new ArrayList<>();
-            topic.configs().forEach((k, v) -> new DescribeConfigsResponse.ConfigEntry(
-                    k, v, source, false, false, Collections.emptySet()
-            ));
-            DescribeConfigsResponse.Config config = new DescribeConfigsResponse.Config(error, entries);
-            configs.put(resource, config);
-        }
-        return new DescribeConfigsResponse(1000, configs);
+            results.add(new DescribeConfigsResponseData.DescribeConfigsResult()
+                    .setErrorCode(error.error().code())
+                    .setErrorMessage(error.message())
+                    .setResourceType(ConfigResource.Type.TOPIC.id())
+                    .setResourceName(topic.name())
+                    .setConfigs(topic.configs().entrySet().stream().map(e -> new DescribeConfigsResponseData.DescribeConfigsResourceResult()
+                            .setName(e.getKey())
+                            .setValue(e.getValue())).collect(Collectors.toList())));
+        }
+        return new DescribeConfigsResponse(new DescribeConfigsResponseData()
+        .setThrottleTimeMs(1000)
```

Comment:
nit: indentation is off here. You may break at the second `new` and have the argument in one line probably